lib/hirefire/macro/sidekiq.rb

```diff
@@ -58,6 +58,35 @@ def queue(*queues)
 
         [in_queues, in_schedule, in_retry, in_progress].compact.inject(&:+)
       end
+
+      # Reports the latency (in seconds) for the provided Sidekiq queue(s).
+      #
+      # @example Sidekiq Macro Usage
+      #   HireFire::Macro::Sidekiq.latency # average latency of all queues
+      #   HireFire::Macro::Sidekiq.queue("email") # only email queue
+      #   HireFire::Macro::Sidekiq.queue("audio", "video") # average of audio and video queue latencies
+      #   HireFire::Macro::Sidekiq.queue("email", "video", weighted: true) # weighted average of latencies
```

Comment:
The last 3 lines of comments invoke the `HireFire::Macro::Sidekiq.queue` function, rather than the `HireFire::Macro::Sidekiq.latency` function.